neqo-http3/src/lib.rs

```diff
@@ -321,6 +323,15 @@ pub trait RecvStream: Debug {
     fn done(&self) -> bool;
     fn stream_type(&self) -> Http3StreamType;
     fn http_stream(&mut self) -> Option<&mut dyn HttpRecvStream>;
+    fn priority(&self) -> Priority;
+
+    /// only relevant for streams supporting priority updates
```

Comment:
I've added some traits that are only relevant for requests and push streams

Reply:
Yea, `HttpRecvStream` looks appropriate for these functions. I even moved priority() there too.